src/components/RadonOperator.vue

```diff
@@ -264,7 +265,6 @@ export default {
           id,
           value: getNativeValueFromMarkupArgumentType(value, type),
         })
-        this.tryDataRequest()
```

Comment:
How this fixes update a variable when is connected to an input and continues updating the partial result?

Reply:
This `tryDataRequest` is redundant. `tryDataRequest` should be called in vuex only